tensorboard/defs/web.bzl

```diff
@@ -99,7 +99,7 @@ def _tf_web_library(ctx):
     )
   params = struct(
       label=str(ctx.label),
-      bind="localhost:0",
+      bind="localhost:6006",
```

Comment:
You mention in your description that the webserver can start with "random ports" but the user can also specify the port with an environment variable. In the future cleanup it would be nice to have this respect the environment variable.

Reply:
I may have missed that: What is the environment variable that users can specify?

Reply:
Discussed offline: We don't believe that we currently offer ability to set the port.